crates/cast/bin/cmd/run.rs

```diff
@@ -52,6 +52,14 @@ pub struct RunArgs {
     #[arg(long, short)]
     verbose: bool,
 
+    /// Prints the trace in JSON format.
+    #[arg(long, short)]
+    json: bool,
+
+    /// Prints the state changes, only works with the `json` flag.
+    #[arg(long)]
+    with_state_changes: bool,
```

Comment:
```suggestion
    /// Prints the state changes.
    #[arg(long)]
    with_state_changes: bool,
```
This should no longer be restricted to `json` only per https://github.com/paradigmxyz/revm-inspectors/pull/213, correct?

Reply:
Updated. thanks!